src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -184,23 +189,29 @@ public void setUp() throws Exception {
         network1 = EurostagTutorialExample1Factory.createWithMoreGenerators(new NetworkFactoryImpl());
         network1.getVariantManager().cloneVariant(VariantManagerConstants.INITIAL_VARIANT_ID, VARIANT_2_ID);
 
+        // Network for nodeBreakerView tests
+        nodeBreakerNetwork = FourSubstationsNodeBreakerFactory.create(new NetworkFactoryImpl());
+        nodeBreakerNetwork.getVariantManager().cloneVariant(VariantManagerConstants.INITIAL_VARIANT_ID, NODE_BREAKER_NETWORK_VARIANT_ID);
+
+        given(networkStoreService.getNetwork(NODE_BREAKER_NETWORK_UUID, PreloadingStrategy.ALL_COLLECTIONS_NEEDED_FOR_BUS_VIEW)).willReturn(nodeBreakerNetwork);
+
         // report service mocking
         doAnswer(i -> null).when(reportService).sendReport(any(), any());
 
         // UUID service mocking to always generate the same result UUID
         given(uuidGeneratorService.generate()).willReturn(RESULT_UUID);
 
         // purge messages
-        while (output.receive(1000, "shortcircuitanalysis.result") != null) {
+        while (output.receive(100, "shortcircuitanalysis.result") != null) {
```

Comment:
why this change ?
probably should be kept as before, these timeouts are finnicky, might be needed by the CI, or someone with a slow PC or whatever